testing/test_aux.out

```diff
@@ -4,15 +4,18 @@ mudacl tests
 Mudacl exit code 0
 Validator exit code 0
 Lint checks
-cmd/inbuild exit code 0
+cmd/inbuild exit code 1
 Extended tests
 RESULT fail network.brute
 RESULT pass network.brute
 RESULT skip network.brute
+RESULT fail security.x509
+RESULT fail security.x509
+]
 dhcp requests 2 1
 01: ['01:brute:1']
 02: ['02:nmap:1']
-03: []
+03: ['03:nmap:1']
```

Comment:
I dont know how this box gets populate [ ]

Reply:
It's from DAQ running the tests, 03 is device 3, nmap is the test, and 1 is the result code. This is likely fine since you added tls to the 03 device, so it's failing nmap because now there's an empty port. This is ok because device 01 still has no nmap (so positive and negative cases).

Reply:
It's because your new addition to container 03 is now "failing" nmap because it has additional ports open that weren't open before.  This isn't a problem since there's still a case (01) that "passes" nmap. So, that's what the